### ISSUE TITLE
Cannot participate in public poll when logged in as guest

### ISSUE THREAD

**Issue by simonspa** [nextcloud/polls#921]
### What is going wrong?
When a user is logged in as guest, they cannot even participate in public polls but are greeted with the following error:

![image](https://user-images.githubusercontent.com/1677436/81384822-854bd600-9112-11ea-81c0-5bf0caa07ca3.png)

**To Reproduce**
1. Get a guest account on nextcloud.com and log in ;)
2. Click on the official Deck survey in the last blog post

**Expected behavior**
Guests should have a higher level of rights than strangers, not a lower. I expect to be able to access and fill the poll despite being logged in as guest and not having full access to the polls app.

(cc) @stefan-niedermann

**Comment by stefan-niedermann** [comment]
I don't think that this is related to the polls app...

**Comment by simonspa** [comment]
Is guest access handled centrally? How would a central instance know what a poll is and whether it has public access rights or not?
I've seen in other apps that they needed to adapt things specifically for the guests app to work properly.

If not, feel free to move.

**Comment by stefan-niedermann** [comment]
@jospoortvliet it's the same issue i told you in talk. @simonspa can not participate at the Deck survey as long as he is logged in as a guest (just like me).

**Comment by dartcafe** [comment]
I would like to help, but I have no idea about which guest login you are talking. Login on nextcloud.com???? Where? I guess the usage of the polls app is limited to user groups.

**Comment by simonspa** [comment]
@dartcafe sorry for the confusion, it was just that I noticed this behavior with a guest account on nextcloud .com. So let me rephrase the "how to reproduce":

* On your NC, install polls and guests apps
* Create new guest account with access e.g. to a single file
* Create a public poll
* Try to access this public poll while being logged in with the guest account

## Expected behavior:
see the poll, be able to answer it - because it's public!

## Actual behavior:
you get the above error because guests don't have access to the resource "polls".

**Comment by dartcafe** [comment]
As far as I see, polls has to get whitelisted in the guests app.

**Comment by simonspa** [comment]
But wouldn't that mean that guest has full access and can use that app?

**Comment by dartcafe** [comment]
> But wouldn't that mean that guest has full access and can use that app?

Yes, see #658, #390

**Comment by wiswedel** [comment]
see https://github.com/nextcloud/guests/issues/380
